app/utils/xpath.py

```
@@ -92,6 +92,17 @@ class Achievements:
 
 
 class Clubs:
+    class Staff:
+        CLUB_NAME = "//header//h1//text()"
+        ID = "//div[@class='large-12 columns']//table[@class='inline-table']//td[@rowspan='2']/img/@src"
+        NAME = "//div[@class='large-12 columns']//table[@class='inline-table']//td[@class='hauptlink']/a/text()"
+        JOB = "//div[@class='large-12 columns']//table[@class='inline-table']//td[@class='hauptlink']/following-sibling::tr[1]/td/text()"
```

Ruff

Line too long (137 > 120)

(E501)